packages/zosfiles/src/methods/create/index.ts

```diff
@@ -10,7 +10,9 @@
 */
 
 export * from "./doc/ICreateDataSetOptions";
+export * from "./doc/ICreateUssOption";
```

Comment:
One of the benefits of splitting API and CLI code into separate packages is that it forces us to make sure we aren't missing any exports 👍